### DISCUSSION TITLE
Certain apps on the Desktop won't show up in the search bar

### DISCUSSION THREAD

**Discussion by Riptidel** [Flow-Launcher/Flow.Launcher#2689] [Q&A] [answered]
I've paired up Flow with the Everything app, and I've noticed that certains apps'shortcuts won't end up in the search bar, but would rather show up some temporary files of said apps.

For example, I've installed "OBS Studio" and "Blender" through Steam, and these two programs are the cause of this issues but the other applications installed by Steam are found easily (always talking about launching the shortcuts on the Desktop).

I've already checked the "Search Full Path" and "Reindex" in the settings, but to no avail.

**Comment by VictoriousRaptor** [comment, marked as answer]
Please add your desktop folder in program plugin.

**Reply by Riptidel** [reply]
Thanks, it works!
I've also discovered that this problem is caused by wrong installation of said apps, since I've checked the the DataProgram's folder for OBS and found out it was empty, but once I've added the desktop shortcut in that folder, Flow would actually find the executable in the search bar.
Probably it's an installation error, but I wanted to point it out just in case there could some automatic workarounds in the future.
